packages/components/src/site-thumbnail/index.tsx

```diff
@@ -0,0 +1,49 @@
+import classnames from 'classnames';
+import { ReactNode } from 'react';
+import './style.scss';
+import { useMshotsImg } from './use-mshots-img';
+import { getTextColorFromBackground } from './utils';
+
+const MSHOTS_OPTION = {
+	vpw: 1200,
+	vph: 1200,
+	w: 374,
```

Comment:
The reason why the image is pixelated is that in retina screens we need at least double the size of the img element.
If the img element will be 374 width, then we need to generate an image of  `w: 374*2`